security/certora/specs/Governance.spec

```diff
@@ -619,6 +665,7 @@ rule immutable_after_creation(method f){
 
 
 // Proposal payloads cannot change.
+/* ND - what about f == createProposal why is this passing */ 
```

Comment:
Invariant empty_payloads_iff_uninitialized_proposal ensures that uninitialized proposals have no payloads.

Reply:
The entire rule passed vacuously and it was removed. Before `createProposal()` the array of payloads is empty `_proposals[proposalId].payloads.length == 0` and we cannot refer to its elements.